src/libNode/Node.cpp

```diff
@@ -239,6 +250,144 @@ void Node::AddGenesisInfo(SyncType syncType) {
   }
 }
 
+bool Node::ValidateDB() {
+  deque<pair<PubKey, Peer>> dsComm;
+  const string lookupIp = "127.0.0.1";
```

Comment:
Why use a local IP?

Reply:
the verifier sends a  message  to inform the lookup machine (running on the same machine) that it has done the verification.